gitian-building.md

```diff
@@ -70,7 +70,7 @@ In order to sign gitian builds on your host machine, which has your PGP key, for
 
 ```
 export NAME=satoshi
-git clone git@github.com:bitcoin-core/gitian.sigs.git
+git clone git@github.com:$NAME/gitian.sigs.git
```

Comment:
NAME is for the name you put in your gpg key. It might be different from the place you forked the repo on github

Reply:
Yes - I agree [in line #74](https://github.com/bitcoin-core/docs/blame/master/gitian-building.md#L74)

But to be clear per the instructions - the user MUST fork ([per line #69](https://github.com/bitcoin-core/docs/blame/master/gitian-building.md#L69)) and clone their own repo to the build machine. After the gitian-build process is complete they push to their own repo and submit a PR to be added to the bitcoin-core/gitian.sigs.git repository for distribution. If we assume the user is following the instructions line by line. Then we should format the command to assume that the user has their own fork that they are cloning to their own build machine.

In line [#74](https://github.com/bitcoin-core/docs/blame/master/gitian-building.md#L74) the first instance of $NAME is referring to a folder that was created during the build process. The second $NAME instance is referring to the users repository. 

LINE 74: git remote add $NAME git@github.com:$NAME/gitian.sigs.git

As in example [https://github.com/bitcoin-core/gitian.sigs/pull/1115](https://github.com/bitcoin-core/gitian.sigs/pull/1115) was pulled from user [Emzy](https://github.com/Emzy/gitian.sigs/tree/0.18.1-not-codesigned)'s fork.
The contents of Emzy's folder within his repo are the products of his gitian build process.


https://github.com/Emzy/gitian.sigs/tree/0.18.1-not-codesigned/0.19.0rc1-win-unsigned/emzy
https://github.com/Emzy/gitian.sigs/tree/0.18.1-not-codesigned/0.19.0rc1-osx-unsigned/emzy etc...